### ISSUE TITLE
Roundrobin sorting option

### ISSUE THREAD

**Issue by KNejad** [FredJul/Flym#750]
It would be great if there was an option to sort the feeds in a Round Robin way.

So basically if I have posts from A, B and C instead of sorting newest to oldest you would sort them like this:

A1, B1, C1, A2, B2, C2...

The reason I would love this feature is because I have some feeds which rarely get published (e.g. Arch Linux news) and some that are filling up my news feed (e.g. Stack Exchange favorite tags). It gets annoying having to go through 5-6 Stack Overflow posts in a row to get to something from another source, and I also never get to see any posts from Arch Linux since they are very far down the list.

With this new sort order I would be able to read posts from all my sources regardless of how many each post.

**Comment by nikhilCad** [comment]
You can add feeds in folders. Swipe left to right, three dots, reorder feeds.

**Comment by KNejad** [comment]
Yeah I'm aware of that feature but I don't think it really solves the issue. I don't want to have to remember to check certain folders and I don't want to be checking say only my politics feeds. Ideally I would like to be on the main page with all the feeds, just without any specific feeds dominating the page. That way I get a bit of everything on the main page
